server/src/main/java/io/spine/server/delivery/LocalDispatchingObserver.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.delivery;
+
+import io.spine.server.ServerEnvironment;
+
+/**
+ * An observer of changes to the shard contents, which triggers immediate delivery of the
+ * sharded messages.
+ *
+ * <p>Suitable for the local and development environment.
+ */
+public class LocalDispatchingObserver implements ShardObserver {
+
+    @Override
+    public void onMessage(InboxMessage update) {
+        Delivery delivery = ServerEnvironment.instance()
+                                             .delivery();
+        delivery.deliverMessagesFrom(update.getShardIndex());
+    }
+}
```

Comment:
Please add the EOF. It seems to be a bug in IDEA. I've already seen this with my own changeset as well.